tools/internal_ci/linux/grpc_e2e_performance_v2.sh

```diff
@@ -29,65 +29,65 @@ gcloud config set project grpc-testing
 gcloud container clusters get-credentials benchmarks-prod \
     --zone us-central1-b --project grpc-testing
 
-# Set up environment variables
-PREBUILT_IMAGE_PREFIX="gcr.io/grpc-testing/e2etesting/pre_built_workers/"${KOKORO_BUILD_INITIATOR}
+# Set up environment variables.
+PREBUILT_IMAGE_PREFIX="gcr.io/grpc-testing/e2etesting/pre_built_workers/${KOKORO_BUILD_INITIATOR}"
 UNIQUE_IDENTIFIER=$(date +%Y%m%d%H%M%S)
 ROOT_DIRECTORY_OF_DOCKERFILES="../test-infra/containers/pre_built_workers/"
 GRPC_CORE_GITREF="$(git ls-remote https://github.com/grpc/grpc.git master | cut -c1-7)"
```

Comment:
since you're running this from a clone of grpc/grpc at a specific commit (likely master, but could be a pull request as well), it makes no sense to ask for the `git ls-remote https://github.com/grpc/grpc.git master` - just use the current local commit, it's more correct and allows you to tests e.g. release branches, pull requests etc.

For grpc-java and grpc-go since we don't have a local clone, it's fine to ask github  the head of master branch. (This is actually very similar to the behavior we already use for interop_tests).

This is fine to address as a followup PR, but it should be addresed.

Reply:
I update to use `git git rev-parse https://github.com/grpc/grpc.git`